src/system.c

```diff
@@ -848,11 +850,11 @@ int lseek( int file, int ptr, int dir )
  * @param[in] flags
  *            Flags specifying open flags, such as binary, append.
  * @param[in] mode
- *            Mode of the file.
+ *            Mode of the file. (TODO: currently ignored)
```

Comment:
idk what to say about this, leave it as a TODO ?

Reply:
I guess you can keep it without TODO:

```suggestion
 *            Mode of the file (currently ignored).
```